docs/getting_started.html

```diff
@@ -0,0 +1,212 @@
+          3. Create/Run Terraform code
+        </a>
+        <a class="list-group-item list-group-item-action" id="list-four-list" data-toggle="list" href="#list-four" role="tab" aria-controls="four">
+          4. Create/Run Inspec Tests
+        </a>
+      </div>
+    </div>
+    <div class="col-8">
+      <div class="tab-content" id="nav-tabContent">
+        <div class="tab-pane fade show active" id="list-one" role="tabpanel" aria-labelledby="list-one-list">
+          Install Terraform: <a href="https://www.terraform.io/downloads.html" style="color: #32c850;">
+            https://www.terraform.io/downloads.html
+          </a>
+          <br>
+          Install Ruby: <a href="https://www.ruby-lang.org/en/documentation/installation/" style="color: #32c850;">
+            https://www.ruby-lang.org/en/documentation/installation/
+          </a>
+          <br><br>
+          Additional installation methods:
+          <br><br>
+          <div class="row">
+            <div class="col-sm">
+              Mac OS (homebrew):
+<div class="highlight"><pre class="syntax-highlight shell"><code>brew install terraform
+brew install ruby
+</code></pre></div>            </div>
+            <div class="col-sm">
+              Version managers:<br>
+              <a href="https://github.com/kamatama41/tfenv" style="color: #32c850;">tfenv</a>
+              <br>
+              <a href="https://github.com/rbenv/rbenv" style="color: #32c850;">rbenv</a>
+            </div>
+          </div>
+        </div>
+        <div class="tab-pane fade" id="list-two" role="tabpanel" aria-labelledby="list-two-list">
+          Create module and folder structure by running these commands:
+          <br><br>
+<div class="highlight"><pre class="syntax-highlight shell"><code>mkdir <span class="nt">-p</span> my_terraform_module <span class="o">&amp;&amp;</span> <span class="nb">cd </span>my_terraform_module
+
+mkdir <span class="nt">-p</span> <span class="nb">test</span>/integration/example/controls <span class="se">\</span>
+         <span class="nb">test</span>/fixtures/example/
+</code></pre></div>         Then install the necessary rubygems.
+         <br><br>
+<div class="highlight"><pre class="syntax-highlight shell"><code><span class="o">(</span>
+<span class="nb">cat</span> <span class="o">&lt;&lt;</span> <span class="no">EOP</span><span class="sh">
+source "https://rubygems.org/" do
+  gem "kitchen-terraform"
+end
+</span><span class="no">EOP
+</span><span class="o">)</span> <span class="o">&gt;</span> Gemfile
+
+<span class="c"># Install kitchen-terraform and other rubygems</span>
+<span class="c"># If needed run `gem install bundler`</span>
+bundle install
+</code></pre></div>          Create this file <p class="font-weight-bold" style="color: #32c850; display: inline;">.kitchen.yml</p> which brings together the Terraform module code and Inspec controls.
```

Comment:
Consider "Create the Test Kitchen configuration file, `.kitchen.yml`, and configure the kitchen-terraform plugins to associate the fixture Terraform module with the InSpec profile." Also, this step should probably be moved to after the actual creation of those components so it's less abstract, or the file contents can be accumulated through each section.

Reply:
Also, let's try to think of other terms than just "example" so the different components are easier to differentiate.

Reply:
I changed the component names, but I think allowing them to copy and paste the entire .kitchen.yml in this quick start guide will be best. I'd also rather keep the .kitchen.yml creation at this point to have them evaluate it now (as nothing else of significance has occurred yet) and as its fresh in the mind go down the rest of the guide.

I've added text to please refer back this as they go on.

Reply:
Rightyo!